api/routers/edr.py

```diff
@@ -80,9 +81,13 @@ async def get_locations(
     ts_response = await get_obs_request(ts_request)
 
     platform_parameters: DefaultDict[str, Set[str]] = defaultdict(set)
+    platform_names: Dict[str, str] = defaultdict(str)
```

Comment:
To check for a feature having more then one platform names (they might be inconsistent in different time series), store them in a `Dict[str, Set[str]]`, and do the same check as for the coordinates?

Reply:
Refactored in https://github.com/EURODEO/e-soh/pull/75/commits/0ef8818d457f7d494cefec908fee2cf05ed4b600